backend/app/model/location_category.py

```diff
@@ -0,0 +1,16 @@
+from app import db
```

Comment:
If there is shared parent class, it might allow us to remove the multiple relationship definitions as well.